File: est datos/c#/Tarea_2.cs
using System;
class Libro
{
    private string title;
    private string author;

    public Libro(string title, string author){
        this.title = title;
        this.author = author;
    }

    public void enseñar(){
        Console.WriteLine($"Título: {this.title}, Autor: {this.author}");
    }
    
};

class Program
{
    static void Main(string[] args)
    { 
        Libro[] biblioteca = new Libro[]
        {
            new Libro("Cien años de soledad", "Gabriel García Márquez"),
            new Libro("El principito", "Antoine de Saint-Exupéry"),
            new Libro("Berserk", "Kentaro Miura")
        };

        foreach (var libro in biblioteca)
        {
            libro.enseñar();
        }
    }

 };

File: est datos/cpp/Tarea_2.cpp
#include <iostream>
#include <string>
using namespace std;

// Definición de la clase Libro
class Libro {
//inicializamos variables en private
    private:
 string titulo;
 string autor;
//métodos en public
 public:
 //constructor
 Libro(string t, string a) : titulo(t), autor(a) {}
    //método para mostrar información del libro
void libritos() {
    cout << "Titulo: " << titulo << ", Autor: " << autor << endl;
}

};

int main() {
    //Crear un arreglo de libritos
    Libro blibliotequita[] = {
        Libro("Berserk", "Kentaro Miura"),
        Libro("EL niño y la garza", "Ghibli"),
        Libro("Cien años de soledad", "Gabriel García Márquez")
    };

    int tam = sizeof(blibliotequita) / sizeof(blibliotequita[0]);

    for (int i = 0; i< tam ; i++){
        blibliotequita[i].libritos();
    };

    return 0;
};